domains/bradley.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Bradley-iii",
+           "email": "",
+           "discord": "852033849567477760",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.jWZUAv5ynU4LXJBAUmta-T6qLqeQy_FowNIvgYK2MLyTyLZzuaUFqjHNYvs1EPpxeQeaxFK2HWSut8sJR6BLgayNjrQufHqb2utoBfptVVcoLQ0bRUjLzxYze6iqDPMOYgy3_YNkOFNkp0Vmb_JoxTAsRv9KvE2aiI9P013c1jbMw9uA4WxYJTf4UHSSxPaZrP7dQmzdrRZKtg0CFrx32B2HTOtMIQkZr5wFC59E4EUjsSEWTBT81I5QjJt7n6TSOwzt6amlP7bQK1AoRB-d8uQHCOXgG5JuvlIqkEoEWP74aA5Im1eYKZzXNp25w5n5GOmU4cFF2Z4xBBs2i8Q1Pw.EAYb4LmZC6Nf49mrfwpE5A.yMuld6tcff0CeqrXdeD5V0SyvqpRnBtsPcz0Wd0YVGTOrFwDdZLylz8cZUhGnqqz-FHeJPIwmSkUj_KQ-MQvrZdqVnYYT0hjdKWsOrFE90A.F2BpJk2GCj_F4f91QjMa7Q"
+        },
+    
+        "record": {
+            "CNAME": "bradley-iii.github.io"
```

Comment:
Is [this](https://github.com/Bradley-iii/Website) your website? if so, you need to complete it first, right now there is just an "Hello World"

Reply:
Yeah I am trying to find my files apologies

Reply:
Update: Lost my old website files nvm